scripts/gemm.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# HACK this does not work with pip-installed benchmarker
+# because binaries are not in site_packages, alse can't use run script
+# as temp solution - this sh script to be copied to root folder manually when needed
```

Comment:
Maybe a bit better explanation/wording?

Reply:
the whole premise of scripts working from scripts folder is that benchmarker should be installed by pip. but that does not copy binary files - which is ok, what I'm saying here is that I've added this script just for myself and put it to scripts folder to look nices, yet I'll cp it to .. when actually running

Reply:
ok. but if I wanna use it, I'm not 100% sure what I should do to run it (I'd probably figure it out but if you add some nicer documentation where to copy/compile what - might make life easier - but not super important)

Reply:
root folder -> benchmarker's root folder?

Reply:
oke... I figured it out... now I can't tell what would be a better wording.. :D but yeah...
maybe (thinking out loud), something like copy it to the root of the benchmarker repo (because "this does not work with pip-installed benchmarker" actually means that you/we would use it in the github repo, right?)